### PR TITLE
Setting up co-anchor on aux chain creations and enabled mosaic verify

### LINKED ISSUE #91: Set CoAnchor address when creating new auxiliary chain
When creating new auxiliary chain, setting up of coAnchor address step is missing in auxiliary and origin chain.

Below method of mosaic.js should be called to set coAnchor address in both origin and auxiliary contract when creating new auxiliary chain.
https://github.com/OpenST/mosaic.js/blob/develop/src/ContractInteract/Anchor.js#L335
```
setCoAnchorAddress(coAnchorAddress, txOptions)
```

Update ChainVerifier to verify coAnchor addresses in both chains.

### PR DESCRIPTION
**Pull request by sarvesh-ost**
fixes #91 
This PR fixes bug in mosaic chain creations and set co-anchor after anchors are deployed.

### CODE REVIEW

File: src/NewChain/OriginChainInteract.ts [hunk 1]
```diff
@@ -211,6 +212,29 @@ export default class OriginChainInteract {
...
+  public get txOptions(): Tx {
+    return this.initConfig.originTxOptions;
+  }
+
+  /**
+   * This method set anchor organization admin.
+   * @param admin Admin address.
+   * @param anchorOrganization AnchorOrganization contract interact.
+   */
+  public async setAnchorOrganizationAdmin(
```
Comment (puneet-khushwani-eth) [inline src/NewChain/OriginChainInteract.ts]:
```suggestion
  public async setOrganizationAdmin(
```

File: src/NewChain/OriginChainInteract.ts [hunk 2]
```diff
@@ -211,6 +212,29 @@ export default class OriginChainInteract {
     return Contracts.deployGatewayLibraries(web3, { from: deployer });
   }
 
+
+  /**
+   * Returns Origin TX Options.
+   */
+  public get txOptions(): Tx {
```
Comment (puneet-khushwani-eth) [inline src/NewChain/OriginChainInteract.ts]:
```suggestion
  private get txOptions(): Tx {
```

File: src/NewChain/Initialization.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { Utils as MosaicUtils } from '@openst/mosaic.js';
+import { Utils as MosaicUtils, ContractInteract } from '@openst/mosaic.js';
```
[the remark this answers is not in the document]
Comment (sarvesh-ost) [reply src/NewChain/Initialization.ts]: Types from mosaic-contracts wont be compatible from mosaic.js contract interacts.

File: src/NewChain/OriginChainInteract.ts [hunk 4]
```diff
@@ -211,6 +212,44 @@ export default class OriginChainInteract {
...
+      this.txOptions,
+    );
+  }
+
+  /**
+   * This method set co-anchor address;
+   * @param originAnchor Instance of anchor contract on origin chain.
+   * @param coAnchorAddress CoAnchor address.
+   */
+  public async setCoanchorAddress(
```
Comment (deepesh-kn) [inline src/NewChain/OriginChainInteract.ts]: Let's keep this consistent with the contract `setCoAnchorAddress `

File: src/NewChain/AuxiliaryChainInteract.ts [hunk 5]
```diff
@@ -289,6 +293,21 @@ export default class AuxiliaryChainInteract {
     this._auxiliarySealer = value;
   }
 
+  /**
+   * This method set anchor organization admin.
+   * @param admin Admin address.
+   * @param anchorOrganization AnchorOrganization contract interact.
+   */
```
[the remark this answers is not in the document]
Comment (sarvesh-ost) [reply src/NewChain/AuxiliaryChainInteract.ts]: Added possible tests.

**Pull request merged**